techniques/html/H28.html

```diff
@@ -97,24 +97,23 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
    
 
    <section id="description">
-      <h2>Description</h2>
-      <p>The objective of this technique is to provide expansions or definitions for
-            abbreviations by using the <code class="language-html">abbr</code> element. It is always appropriate to use the <code class="language-html">abbr</code> element for any abbreviation, including acronyms and initialisms.</p>
-            <p>This technique has been altered from a sufficient technique to an advisory technique because it uses the <code>title</code> attribute of <code>abbr</code> to provide the expansion of the abbreviation needed to achieve a sufficient technique when used in combination with <a href="../general/G102">G102</a>. As noted in the <a href="https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute"> specification</a>:</p>
+      <h2>解説</h2>
+      <p>このテクニックの目的は、<code class="language-html">abbr</code> 要素を用いて略語の展開形又は定義を提供することである。頭字語及び頭文字を含むあらゆる略語には、常に <code class="language-html">abbr</code> 要素を使用することが常に適切である。</p>
+            <p>このテクニックは、<a href="../general/G102">G102</a> と組み合わせて使用した場合に十分なテクニックを達成するために必要な略語の展開形を提供するために abbr の title 属性を使用しているため、十分なテクニックから参考テクニックに変更された。<a href="https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute"> 仕様書</a>には次のように記載されている:</p>
```

Comment:
「ために」が続いて読み取りにくい感じがしましたので、訳を調整してみました。いかがでしょうか？
```suggestion
            <p>このテクニックは、<a href="../general/G102">G102</a> と組み合わせて使用した場合に十分なテクニックを達成するのに必要な、略語の展開形を提供するために <code>abbr</code> の <code>title</code> 属性を使用していることから、十分なテクニックから参考テクニックに変更された。<a href="https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute"> 仕様書</a>には次のように記載されている:</p>
```